arduino_output/OdomRaw.msg

```diff
@@ -0,0 +1,11 @@
+# Message to transport raw odometry information from
+# the Arduino microcontroller to the main ROS system.
+# It is expected that this raw data will be expanded
+# into the full odometry messages used by ROS.
+
+Header header
+float64 x
```

Comment:
The savings comes from that `nav_msgs/Odometry` 
http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html
Contains these guys:
http://docs.ros.org/api/geometry_msgs/html/msg/PoseWithCovariance.html
http://docs.ros.org/api/geometry_msgs/html/msg/TwistWithCovariance.html
Each containing 36 `float64` saving 72 `float64`'s from being sent.